module_prototyper/models/module_prototyper.py

```diff
@@ -0,0 +1,450 @@
+# -*- encoding: utf-8 -*-
+# #############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2010 - 2014 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import base64
+import lxml.etree
+import os
+import re
+import textwrap
+
+from collections import namedtuple
+from datetime import date
+
+from jinja2 import Environment, FileSystemLoader
+
+from openerp import models, api, fields
+
+from .default_description import get_default_description
+
+
+class ModulePrototyper(models.Model):
+    """Module Prototyper gathers different information from all over the
+    database to build a prototype of module.
+    We are calling it a prototype as it will most likely need to be reviewed
+    by a developer to fix glitch that would sneak it during the generation of
+    files but also to add not supported features.
+    """
+    _name = "module_prototyper"
+    _description = "Module Prototyper"
+
+    license = fields.Selection(
+        [('GPL-2', 'GPL Version 2'),
```

Comment:
I'd remove GPL-2 strict from the list of proposed licenses, as it is incompatible with AGPL-3 used by the core and most OCA addons. Maybe add LGPL-3 instead.